CHANGELOG.md

```diff
@@ -2,6 +2,13 @@
 All notable changes to this project will be documented in this file.
 This project adheres to [Semantic Versioning](http://semver.org/).
 
+## [1.6.1](https://github.com/sonata-project/SonataArticleBundle/compare/1.6.0...1.6.1) - 2021-07-17
+### Changed
+- [[#280](https://github.com/sonata-project/SonataArticleBundle/pull/280)] JS will now use the delete field to mark a fragment as deleted instead of not sending the value. ([@srascar](https://github.com/srascar))
```

Comment:
```suggestion
- [[#280](https://github.com/sonata-project/SonataArticleBundle/pull/280)] JS will now use the delete field to mark a fragment as deleted instead of not sending the value ([@srascar](https://github.com/srascar))
```